### PR TITLE
feat: replace template paths with contents

### PR DESCRIPTION
**Pull request by Shou**
We replace the `templateUrl` fields with `template` fields and thus
switch from template paths to template contents in preparation for the
Webpack PR.

Changelog-Entry: Replace template paths with template contents.
Change-Type: patch

### CODE REVIEW

**Comment by lurch** [comment]
Picking up the changes from https://github.com/resin-io/etcher/pull/1943/files#diff-be8e7cbae4a85d432ac97a2164bcef95 might allow the CI tests to pass ;-)

File: tests/gui/pages/main.spec.js [hunk 1]
```diff
@@ -241,4 +250,22 @@ describe('Browser: MainPage', function () {
...
   })
+
+  describe('page template', function () {
+    let $state
+
+    beforeEach(angular.mock.inject(function (_$state_) {
+      $state = _$state_
+    }))
+
+    it('should match the file contents', function () {
```
Comment (lurch) [inline tests/gui/pages/main.spec.js]: Is this also a temporary thing, or will it remain after webpack has been integrated?

Comment (Shou) [reply tests/gui/pages/main.spec.js]: What's in the test file should remain as-is with Webpack. We will only remove the custom HTML `request` handler outside `tests/` as Webpack takes care of that.

Comment (lurch) [reply tests/gui/pages/main.spec.js]: In that case is it also worth testing the other template files? Or is it safe to assume that as long as one got loaded okay, then all the others will have got loaded okay?

**Comment by Shou** [comment]
@lurch thanks! Here it is with the sanity script changes.

File: tests/gui/pages/main.spec.js (hunk 1 above)
Comment (Shou) [reply tests/gui/pages/main.spec.js]: Yeah that's a good idea, we could certainly test the other pages in the same manner, we just need to create test files for them. Hopefully there's a nice way to test the modals too.

**Comment by Shou** [comment]
I've added tests for other pages, unfortunately I can't find a nice way to test the modal(s), but I did add some tests related to the template in any case. Let me know what you think.

**Comment by Shou** [comment]
Tests pass here now, let me know what you think.

File: tests/gui/pages/main.spec.js [hunk 2]
```diff
@@ -26,6 +27,14 @@ const availableDrives = require('../../../lib/shared/models/available-drives')
 const selectionState = require('../../../lib/shared/models/selection-state')
 require('angular-mocks')
 
+// Mock HTML requires by reading from the file-system
+// eslint-disable-next-line node/no-deprecated-api
+require.extensions['.html'] = (module, filename) => {
```
Comment (lurch) [inline tests/gui/pages/main.spec.js]: Out of curiosity, why is this snippet needed in `main.spec.js` but not in `finish.spec.js` or `settings.spec.js`?

**Pull request merged**